### ISSUE TITLE
Revise Package Summary?

### ISSUE THREAD

**Issue by pearsonca** [epinowcast/epinowcast#393]
As part revising the README for more concise language, I proposed an overhaul to the package summary there:

```
_Nowcasting_ is predicting the *current* situation, addressing the practical reality
that the data we want *now* will not be available until *later*. The `{epinowcast}`
package provides you tools for nowcasting epidemiological data in a convenient,
flexible, and efficient collection of `R` functions. Those functions implement a model
of infectious disease spread and reporting using a semi-mechanistic Bayesian
approach with a range of reporting and generative processes. Using this approach,
you will typically see changes in trends earlier and more reliably than if using
methods like truncation.
```

(there's also a note about other applications, but I've moved that to an aside). For reference, the summary is currently (in the README and DESCRIPTION:

```
Tools to enable flexible and efficient hierarchical nowcasting of right-truncated
epidemiological time-series using a semi-mechanistic Bayesian model with support
for a range of reporting and generative processes. Nowcasting, in this context, is
gaining situational awareness using currently available observations and the reporting
patterns of historical observations. This can be useful when tracking the spread of
infectious disease in real-time: without nowcasting, changes in trends can be
obfuscated by partial reporting or their detection may be delayed due to the use of
simpler methods like truncation. While the package has been designed with
epidemiological applications in mind, it could be applied to any set of right-truncated
time-series count data.
```

For the purposes of revising the README, I focused on plainer language and orienting it towards what-we-do-for-you-sers.

@seabbs asked I partition that change out for broader discussion.

A few thoughts:
 - the README and DESCRIPTION summaries serve different audiences, so they needn't be the same in my opinion
 - however, I am sympathetic to making them same (~~though if that's what we're going to commit to, the README should just load the DESCRIPTION field, which we can do easily enough~~ - _done_)
 - whichever summary we're talking about, there is an element of both presenting something for people to read and using keywords for spiders to read. The proposed revision is definitely more implicit on some elements (e.g. "situational awareness" and "historical observations") 
 - the proposed revision is definitely within the universe of README summaries (formatting characters aside). E.g. ggplot is `A system for 'declaratively' creating graphics, based on "The Grammar of Graphics". You provide the data, tell 'ggplot2' how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.`

Please chime in with your thoughts.

**Comment by seabbs** [comment]
> owever, I am sympathetic to making them sam

Yeah I think we want to keep them the same to just reduce the amount of maintenance we have to do. 

I like the more relaxed style of the suggested update but I think we might want to take this chance to expand the scope of the summary to reflect the idea that the package is a flexible tool for answering a range of real-time analysis questions (i.e. beyond nowcasting) or will be. 

For each specific (i.e. nowcasting, and reproduction number estimation at the moment) we could then have drop down sections in the README?

**Comment by pearsonca** [comment]
Fair re updating the summary/description, but: what is that feature set, precisely?

Agree re general update to the README to reflect those different activities, but I'd say the better effort there is on organizing the epinowcast.Rmd => getting-started-nowcasting, getting-started-reff-est, ... => advanced-nowcasting, advanced-... vignette pathway.

Way forward roughly coalescing to me as:
 - [ ] get specific on what epinowcast "does" now (in this thread works)
 - [ ] translate that into user-friendly lingo, update DESCRIPTION => update README (auto now) (PR 1)
 - [ ] bang out getting-started-X vignettes for each "bullet point" feature (PRs 2- n+1, n == number of features)
 - [ ] rearrange epinowcast package vignette (aka, intro) to be an even-quicker-version of each feature (basically, a check on "have i installed the package correctly?"), then point people to the getting-started-X series (PR n+2)
 - [ ] pencil out the advanced-X vignettes (a new issue)
